c5g7/openmc/2d/build-xml-2d.py

```diff
@@ -47,30 +44,26 @@
 lattices['Core'].universes = [[u, m, w], [m, u, w], [w, w, w]]
 cells['Core'].fill = lattices['Core']
 
-# Instantiate a Geometry and register the root Universe
+# Instantiate a Geometry, register the root Universe, and export to XML
 geometry = openmc.Geometry()
 geometry.root_universe = universes['Root']
-
-# Instantiate a GeometryFile, register Geometry, and export to XML
-geometry_file = openmc.GeometryFile()
-geometry_file.geometry = geometry
-geometry_file.export_to_xml()
+geometry.export_to_xml()
 
 
 ###############################################################################
 #                   Exporting to OpenMC settings.xml File
 ###############################################################################
 
-# Instantiate a SettingsFile, set all runtime parameters, and export to XML
-settings_file = openmc.SettingsFile()
+# Instantiate a Settings, set all runtime parameters, and export to XML
+settings_file = openmc.Settings()
 settings_file.energy_mode = "multi-group"
 settings_file.cross_sections = "./mg_cross_sections.xml"
```

Comment:
This filename should be mgxs.xml now after we changed the default filename in OpenMC